packages/shared/src/components/profile/PostsSection.tsx

```diff
@@ -53,6 +56,43 @@ const PostStat = classed(
 );
 const postStatIconClass = 'icon mr-1 text-xl';
 
+const iconImageClass =
+  'top-1/2 left-0 w-8 h-8 bg-theme-bg-primary rounded-full -translate-x-1/2 -translate-y-1/2';
+
+const iconImage = (post: Post) => {
+  if (post.isAuthor || post.isScout) {
+    return (
+      <div
+        className={classNames(
+          iconImageClass,
+          'absolute flex items-center justify-center',
+        )}
+      >
+        {post.isAuthor ? (
```

Comment:
I think we can just use the `authorId` property to compare right? Since if the user did scout an article, its `authorId` will either be if it was anyone's article or it will be empty?

Reply:
This is related to which users profile you are viewing, that's why I had to introduce the two selects as well.
Happy to chat on gather so we can close these?